marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -102,7 +102,7 @@ data PartyAction
              | WaitForTimeout Slot
              | WaitOtherActionUntil Slot
              | NotSure
-             | CloseContract
+             | CloseContract [Payment]
```

Comment:
this is only needed for the `auto` endpoint, right?

Reply:
Correct